.github/workflows/build-plugin-zip.yml

```diff
@@ -1,7 +1,15 @@
 name: Build Plugin Zip
 
 on:
-  workflow_call:
```

Comment:
Do we want to maintain the ability to call the build ZIP action manually from the Actions page (e.g. someone wanting to create a specific build from a specific branch)?

Reply:
No need, was more curiosity, fine to proceed without.